app/tests/evaluation_tests/test_views.py

```diff
@@ -587,10 +587,26 @@ def test_hidden_phase_visible_for_admins_but_not_participants(client):
             reverse_kwargs={"challenge_short_name": ch.short_name, **kwargs},
             user=u,
         )
+
+        if view_name in [
```

Comment:
If statements should be avoided in tests. You should add this to the `view_name, kwargs, status, icon` parameterisation above (the for loop is also not a good pattern, and should be refactored to use a pytest parameterisation + a fixture instead but don't bother with that as part of this PR).

Reply:
Thanks for the feedback. I will adhere to it in the next PRs

Reply:
btw, this test is not written by me. I am modifying it since the test failed after I adapted the view. I indeed used parameterisation in the tests I wrote ([here](https://github.com/comic/grand-challenge.org/blob/4e30279808d0c730398e898705e0addf32a0a0f5/app/tests/evaluation_tests/test_models.py#L691)).

Reply:
I know, but you still added the if statement which is what my comment was about.

Reply:
Indeed! I will avoid it in next PRs
Thanks!!